### ISSUE TITLE
[Docs] Clarify licensing information for bigquery public dataset

### ISSUE THREAD

**Issue by sriumcp** [ossf/scorecard#4143]
**Is your feature request related to a problem? Please describe.**
The README describes scorecard data that is available through a public BigQuery dataset. But the licensing info for this dataset is not explicitly described.

**Describe the solution you'd like**
Please clarify the licensing info for this dataset in the README.

**Additional context**
The data returned by the REST API is available through the CDLA Permissive 2.0 license as described [here](https://cdla.dev/permissive-2-0). If I were to guess, the BigQuery public dataset, which can be queried without the REST API (e.g., using the `bq` CLI) is also available under the same license terms. It would be nice to see this clarified.

**Comment by raghavkaul** [comment]
Possible duplicate of #2713, unless this is specifically about BigQuery data and not API data.

cc @david-a-wheeler

**Comment by sriumcp** [comment]
> Possible duplicate of https://github.com/ossf/scorecard/issues/2713, unless this is specifically about BigQuery data and not API data.

Yes, this is specifically about the BigQuery data. The API data licensing is already documented.

**Comment by shouryathakur-sorcerer** [comment]
Hi! I’d like to work on this issue by clarifying the licensing information for the BigQuery public dataset in the README. Please let me know if that sounds good.
